### PR TITLE
Adds server hardware utilization support

### PR DESCRIPTION
**Pull request by GustavoHennig**
Implements server-hardware/{id}/utilization endpoint;
Small refactoring moving: get_utilization implementation to ResourceClient

### CODE REVIEW

File: examples/server_hardware.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# Try load config from a file (if there is a config file)
+config = try_load_from_file(config)
+
+oneview_client = OneViewClient(config)
+
+# Get Statistics with defaults
+print("Get server-hardware statistics")
+try:
+    switch_statistics = oneview_client.server_hardware.get_utilization("37333036-3831-584D-5131-303030333037")
```
Comment (danielreed) [inline examples/server_hardware.py]: seems like a better name would be server_utilization

File: examples/server_hardware.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    pprint(switch_statistics)
+except HPOneViewException as e:
+    print(e.msg['message'])
+
+
+# Get Statistics specifying parameters
+print("Get server-hardware statistics specifying parameters")
+try:
+    switch_statistics = oneview_client.server_hardware.get_utilization("37333036-3831-584D-5131-303030333037",
```
Comment (danielreed) [inline examples/server_hardware.py]: maybe switch_statistics should be server_utilization

File: examples/server_hardware.py (hunk 1 above)
Comment (GustavoHennig) [inline examples/server_hardware.py]: Sure, you are right. This was a copy and paste issue.

File: examples/server_hardware.py (hunk 2 above)
Comment (GustavoHennig) [inline examples/server_hardware.py]: I'm going to fix it. Thanks.

File: examples/server_hardware.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    }
+}
+
+# Try load config from a file (if there is a config file)
+config = try_load_from_file(config)
+
+oneview_client = OneViewClient(config)
+
+# Get Statistics with defaults
+print("Get server-hardware statistics")
```
Comment (balestrinc) [inline examples/server_hardware.py]: Wouldn't be better "Get server hardware utilization with defaults" ?

Comment (GustavoHennig) [inline examples/server_hardware.py]: Perhaps, but so we should change all other examples too, they are similar. What do you think?

Comment (balestrinc) [inline examples/server_hardware.py]: You know, it's just a detail ..

**Pull request merged**